### PR TITLE
Release/0.13.0

### LINKED ISSUE #84: Discussion on next release
There are a number of changes that have been completed:

https://github.com/nasa/python_cmr/blob/develop/CHANGELOG.md#unreleased

Suggestion is to create a release containing all of these changes. I think it makes sense for the next release to be a minor release as 0.13.0.

@nasa/python_cmr any objections?

### PR DESCRIPTION
**Pull request by frankinspace**
Closes #84 

Release candidate is available for verification 

https://pypi.org/project/python-cmr/0.13.0rc1/
https://github.com/nasa/python_cmr/releases/tag/v0.13.0-rc1

### CODE REVIEW

File: cmr/queries.py [hunk 1]
```diff
@@ -339,18 +391,18 @@ def online_only(self, online_only: bool = True) -> Self:
         return self
 
     def _format_date(
-        self,
-        date_from: Optional[DateLike],
-        date_to: Optional[DateLike]
+            self,
+            date_from: Optional[DateLike],
+            date_to: Optional[DateLike]
```
Comment (chuckwondo) [inline cmr/queries.py:396]: I think extra indentation slipped in via another PR:

```suggestion
        self,
        date_from: Optional[DateLike],
        date_to: Optional[DateLike]
```

File: cmr/queries.py [hunk 2]
```diff
@@ -390,14 +442,14 @@ def convert_to_string(date: Optional[DateLike], default: datetime) -> str:
...
 
     def revision_date(
-        self,
-        date_from: Optional[DateLike],
-        date_to: Optional[DateLike],
-        exclude_boundary: bool = False,
+            self,
+            date_from: Optional[DateLike],
+            date_to: Optional[DateLike],
+            exclude_boundary: bool = False,
```
Comment (chuckwondo) [inline cmr/queries.py:452]: I think extra indentation slipped in via another PR:

```suggestion
        self,
        date_from: Optional[DateLike],
        date_to: Optional[DateLike],
        exclude_boundary: bool = False,
```

File: cmr/queries.py [hunk 3]
```diff
@@ -427,10 +479,10 @@ def revision_date(
...
 
     def temporal(
-        self,
-        date_from: Optional[DateLike],
-        date_to: Optional[DateLike],
-        exclude_boundary: bool = False,
+            self,
+            date_from: Optional[DateLike],
+            date_to: Optional[DateLike],
+            exclude_boundary: bool = False,
```
Comment (chuckwondo) [inline cmr/queries.py:485]: I think extra indentation slipped in via another PR:

```suggestion
        self,
        date_from: Optional[DateLike],
        date_to: Optional[DateLike],
        exclude_boundary: bool = False,
```

File: cmr/queries.py [hunk 4]
```diff
@@ -562,11 +622,11 @@ def polygon(self, coordinates: Sequence[PointLike]) -> Self:
...
-        self,
-        lower_left_lon: FloatLike,
-        lower_left_lat: FloatLike,
-        upper_right_lon: FloatLike,
-        upper_right_lat: FloatLike,
+            self,
+            lower_left_lon: FloatLike,
+            lower_left_lat: FloatLike,
+            upper_right_lon: FloatLike,
+            upper_right_lat: FloatLike,
```
Comment (chuckwondo) [inline cmr/queries.py:629]: I think extra indentation slipped in via another PR:

```suggestion
        self,
        lower_left_lon: FloatLike,
        lower_left_lat: FloatLike,
        upper_right_lon: FloatLike,
        upper_right_lat: FloatLike,
```

File: cmr/queries.py [hunk 5]
```diff
@@ -668,7 +742,7 @@ def __init__(self, mode: str = CMR_OPS):
         self.concept_id_chars = {"G", "C"}
 
     def orbit_number(
-        self, orbit1: FloatLike, orbit2: Optional[FloatLike] = None
+            self, orbit1: FloatLike, orbit2: Optional[FloatLike] = None
```
Comment (chuckwondo) [inline cmr/queries.py:745]: I think extra indentation slipped in via another PR:

```suggestion
        self, orbit1: FloatLike, orbit2: Optional[FloatLike] = None
```

File: cmr/queries.py [hunk 6]
```diff
@@ -832,7 +892,7 @@ def granule_ur(self, granule_ur: str) -> Self:
         return self
 
     def readable_granule_name(
-        self, readable_granule_name: Union[str, Sequence[str]]
+            self, readable_granule_name: Union[str, Sequence[str]]
```
Comment (chuckwondo) [inline cmr/queries.py:895]: I think extra indentation slipped in via another PR:

```suggestion
        self, readable_granule_name: Union[str, Sequence[str]]
```

File: README.md [hunk 7]
```diff
@@ -220,6 +225,9 @@ api.name('/AMR_Side_1/acc_lat')
 
 # Search via concept_id
 api.concept_id('V2112019824-POCLOUD')
+
+# Search via instance format
+query.instance_format(["zarr", "kerchunk"])
```
Comment (chuckwondo) [inline README.md:230]: Should `query` be `api` here, to match everything above?

```suggestion
api.instance_format(["zarr", "kerchunk"])
```

**Review by chuckwondo — APPROVED** [review]
@frankinspace, I made some very minor suggestions, but nothing blocking, so I'm approving -- feel free to take my suggestions or leave them

**Comment by frankinspace** [comment]
Yeah I auto-formatted using PyCharm to help clear some linting errors. Seems like it over-indented in some places

**Comment by chuckwondo** [comment]
No worries. I thought they came from one of the PRs in the bunch. Do we want to create an issue for updating our process to use `ruff`? (We might need to do some reformatting to deal with long lines and perhaps a few other things that we currently let flake8 flag without failing the build.)

**Comment by frankinspace** [comment]
I haven't used `ruff` yet but yeah I'm not tied to flake8 as a linter... as long as we're running a linter I'm happy

**Pull request merged**